pallene/parser.lua

```diff
@@ -277,6 +222,9 @@ function Parser:SimpleType()
 
     elseif self:peek("NAME") then
         local tok = self:e()
+        if tok.value == 'module' then
+            return ast.Type.Module(tok.loc)
+        end
```

Comment:
I'll try option 2

Reply:
I'm answering here because github wouldn't let me answer as a reply to the original comment (https://github.com/pallene-lang/pallene/pull/361#discussion_r617976829).
Yes, we can delete `ast.toplevel_names`.
Sorry, I thought I had deleted it already.